test/agentchat/contrib/vectordb/test_pgvectordb.py

```diff
@@ -24,12 +26,52 @@
     reason=reason,
 )
 def test_pgvector():
```

Comment:
This is unrelated to this PR, but do you have experience with testcontainers? https://testcontainers.com/. It's a pretty cool way to test code with databases.

Reply:
I wasn't aware of this bit of tech. Let me dig into it this weekend. Thank you for sharing!